sentry/src/main/java/io/sentry/util/SampleRateUtil.java

```diff
@@ -28,6 +28,19 @@ public static boolean isValidTracesSampleRate(
       return allowNull;
     }
 
-    return !(tracesSampleRate.isNaN() || (tracesSampleRate > 1.0 || tracesSampleRate < 0.0));
+    return !(tracesSampleRate.isNaN() || tracesSampleRate > 1.0 || tracesSampleRate < 0.0);
+  }
+
+  public static boolean isValidProfilesSampleRate(@Nullable Double profilesSampleRate) {
+    return isValidProfilesSampleRate(profilesSampleRate, true);
+  }
+
+  public static boolean isValidProfilesSampleRate(
+      @Nullable Double profilesSampleRate, boolean allowNull) {
```

Comment:
Why do we need `allowNull`?

Reply:
was for consistency, but it actually doesn't make any sense.
removing it